test/EFCore.PG.FunctionalTests/Query/ListQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
+using Xunit;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    /// <summary>
+    /// Provides unit tests for list operator translations.
+    /// </summary>
+    public class ListQueryNpgsqlTest : IClassFixture<ListQueryNpgsqlTest.ListQueryNpgsqlFixture>
```

Comment:
You may want to consider having a single test suite for both array and list. After all, the operations are logically the same and we map to the same PostgreSQL type. Let me know what you think.